Configurable aliases

RenderBr opened the pull request:
Aliases are now defined in `Configuration/Models/Aliases.cs`, this is deserialized/serialized inside Configuration.js, and contains default values for command aliases. They can be configured in the JSON config, and should apply dynamically on reload. 

This allows for:
- Configurable alias names
- Configurable amount of aliases
- Aliases do not have to be included in lang files since the user has ultimate control over them
- Can be dynamically reloaded
- Aliases no longer need to be defined in Command classes, as they will be automatically retrieved using `CommandsBase.GetAliases()`

Maxthegreat99 committed:
most if not all strings are now localized

also changed our working directory

RenderBr committed:
configurable aliases + cleanup cmds

RenderBr committed:
minor oversight - dynamic reloading works

RenderBr committed:
fixed exception when no aliases are provided

Maxthegreat99 reviewed (approved):
Everything seems good, i just need to actually remove aliases from the lang file then add a bunch more stuff regarding localization then we can merge it i think. good job man.

RenderBr committed:
minor changes

RenderBr committed:
typo fix

RenderBr committed:
Create PetaPoco.dll

Maxthegreat99 commented on TSEconomy/Api/Api.cs:
```diff
@@ -13,16 +12,24 @@ public static class Api
+        /// </summary>
+        private static IDatabase DB => TSEconomy.DB.DB;
+        
+        // ? why are we splitting these?
```
thought of it as a way for other plugins to add non-configs currencies. Those arent saved so i think it would be better to make change it so that we are saving currencies to the DB wed need to make sure we dont want to save the configs currencies tho right?

Maxthegreat99 commented on TSEconomy/Helpers.cs:
```diff
@@ -17,14 +31,15 @@ public class Helpers
             return null;
         }
 
+        // ?? is this necessary when TShock.UserAccounts.GetUserAccountByID exists
```
yes because the server has no user account, since we are storing the world acc's ID as -1 its name should be Server. can try to find a better alternative since you cant actually get a userAcc from the world account's ID in the first place

Maxthegreat99 committed:
Merge branch 'main' into aliases-configurable

Maxthegreat99 commented on TSEconomy/Database/Models/BankAccount.cs:
```diff
@@ -29,8 +29,9 @@ public class BankAccount
 
         [Column("Balance")]
+
         public double Balance { get { return _balance; } private set { } }
```
fixed merge conflict

Maxthegreat99 committed:
cleanup, added russian localization, removed ...

... command aliases / names from the plugin's localized texts, the path for localized texts are now configurable, the plugin now loads its default language depending on what language the server is running on, defaults to english if the language is unsupported

Maxthegreat99 committed:
cleanup, fixed naming, no longer serializing an...

... internal property

Maxthegreat99 committed:
fixed convention, actually added the russian ...

localized texts as embed resource

Maxthegreat99 committed:
clean up, fix concention
